packages/cli/src/lib/project/manifests/app/load.ts

```diff
@@ -6,7 +6,11 @@ import {
 } from "@polywrap/polywrap-manifest-types-js";
 import fs from "fs";
 
-export const defaultAppManifest = ["polywrap.app.yaml", "polywrap.app.yml"];
+export const defaultAppManifest = [
+  "polywrap.yaml",
+  "polywrap.app.yaml",
+  "polywrap.app.yml",
```

Comment:
Should we still support this naming? :thinking:

Reply:
for legacy sake, sure, but not going forward :P